test_data.json

```diff
@@ -0,0 +1,191 @@
+[
```

Comment:
Can anyone think of reasons to change this json format that would make using it easier in the other SDKs?